src/styles/blocks/cards.scss

```
@@ -0,0 +1,138 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.cards__card {
```

```suggestion
&__card {...}
```
you need to apply SASS Nesting